pkg/uiserver/uiserver.go

```diff
@@ -53,6 +66,61 @@ func RewriteAssets(fs http.FileSystem, cfg *config.Config, updater UpdateContent
 
 	rewrite("/index.html")
 	rewrite("/diagnoseReport.html")
+
+	overrideDistroAssetsRes(fs, distroResFolderPath, updater)
+}
+
+func overrideDistroAssetsRes(fs http.FileSystem, distroResFolderPath string, updater UpdateContentFunc) {
+	info, err := os.Stat(distroResFolderPath)
+	if errors.Is(err, os.ErrNotExist) || !info.IsDir() {
+		// just ignore if the folder doesn't exist or it's not a folder
+		return
+	}
+	if err != nil {
+		log.Fatal("Failed to stat dir", zap.String("dir", distroResFolderPath), zap.Error(err))
```

Comment:
Have you confirmed that crash is the accepted behavior?

Reply:
No reply yet, I will DM alex soon.